.buildkite/pipelines/flaky_tests/groups.json

```diff
@@ -13,6 +13,11 @@
       "key": "oss/accessibility",
       "name": "OSS Accessibility"
     },
+    {
+      "key": "xpack/cypress",
+      "name": "Default Cypress",
+      "ciGroups": 3
+    },
```

Comment:
```suggestion
    {
      "key": "xpack/cypress-security_solution",
      "name": "Security Solution - Cypress"
    },
    {
      "key": "xpack/cypress-osquery_cypress",
      "name": "Osquery - Cypress"
    },
    {
      "key": "xpack/cypress-fleet_cypress",
      "name": "Fleet - Cypress"
    },
```

Reply:
Yep, agreed. This makes more sense. Making the change